pkg/deps/config/suppliers.go

```diff
@@ -69,7 +69,7 @@ func NewSupplyEventsQueryClientFn(queryNodeRPCURL *url.URL) SupplierFn {
 		_ *cobra.Command,
 	) (depinject.Config, error) {
 		// Convert the host to a websocket URL
-		queryNodeWebsocketURL := sdk.HostToWebsocketURL(queryNodeRPCURL.Host)
+		queryNodeWebsocketURL := sdk.HostToWebsocketURL(queryNodeRPCURL)
```

Comment:
Please rename to something like `RPCToWebsocketURL` since we are no longer passing `.Host`